alphabase/spectral_library/base.py

```diff
@@ -135,9 +136,9 @@ def fragment_intensity_df(self)->pd.DataFrame:
         return self._fragment_intensity_df
     
 
-    def available_fragment_dfs(self)->list:
+    def available_dense_fragment_dfs(self)->list:
```

Comment:
What about move this into flat_lib? As a base lib must have dense frag dfs

Reply:
It's only applicable to the dense lib. The flat lib will always return an empty list.

The issue is that it's inherited.